docs/user/OsmApiDbConverter.asciidoc

```diff
@@ -22,32 +22,32 @@ services.db.writer.output.id.mappings=/tmp/idmaps.xml  e.g. optional to validati
 a sample osm file: $HOOT_HOME/test-files/ToyTestA.osm
 
 % Output source:
-a sample OpenStreetMap Editing API Database: postgresql://hoot:hoottest@localhost:5432/osmapi_test  This database is the test database and should be active for testing. However, any database output source may be applied here. 
+a sample OpenStreetMap Editing API Database: osmapidb://hoot:hoottest@localhost:5432/osmapi_test  This database is the test database and should be active for testing. However, any database output source may be applied here. 
 
 % Command syntax example:
-hoot convert -D services.db.writer.email=OsmApiInsert@hoot.local -D services.db.writer.create.user=true -D reader.add.source.datetime=false -D services.db.writer.output.id.mappings=/tmp/idmaps.xml $HOOT_HOME/test-files/ToyTestA.osm postgresql://hoot:hoottest@localhost:5432/osmapi_test
+hoot convert -D services.db.writer.email=OsmApiInsert@hoot.local -D services.db.writer.create.user=true -D reader.add.source.datetime=false -D services.db.writer.output.id.mappings=/tmp/idmaps.xml $HOOT_HOME/test-files/ToyTestA.osm osmapidb://hoot:hoottest@localhost:5432/osmapi_test
 ------
 
+NOTE: Currently it is not possible to write to an OpenStreetMap API database so this example will likely be updated in future versions.
```

Comment:
I think we should probably just remove the example, but keep a note that writing isn't supported at this time.